nextcord/message.py

```diff
@@ -543,6 +545,56 @@ def flatten_handlers(cls):
     return cls
 
 
+class MessageInteraction:
```

Comment:
Similarly, as this has a Discord id, we can have it implement Hashable.
```suggestion
class MessageInteraction(Hashable):
```

Reply:
Also gotta add the .created_at to the docs

Reply:
The docs should keep track of properties like that automatically